### PR TITLE
Improve PR template

### PR DESCRIPTION
**Pull request by iizukak**
Solve #814

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,31 +1,2 @@
...
-## Checklist:
-<!--- ✅✅✅ Go over all the following points, and put an `x` in all the boxes that apply. ✅✅✅ -->
-<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
-
-<!--- TODO(wakisaka): After decide our code style, add this.
-- [ ] My code follows the code style of this project.
--->
-- [ ] My change requires a change to the documentation.
-- [ ] I have updated the documentation accordingly.
+Dcribe what this patch does to fix the issue.
```
Comment (yd8534976) [inline .github/PULL_REQUEST_TEMPLATE.md]: Thanks! Please fix typo. And I think we'd better highlight this line as below.
## Describe what this patch does to fix the issue.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,31 +1,2 @@
...
+Link to any relevant issues or pull requests.
```
Comment (yd8534976) [inline .github/PULL_REQUEST_TEMPLATE.md]: These two lines will be in one line if no space line.

Comment (iizukak) [reply .github/PULL_REQUEST_TEMPLATE.md]: @yd8534976 Wow, fixed. How about?

Comment (yd8534976) [reply .github/PULL_REQUEST_TEMPLATE.md]: @iizukak  Thanks! LGTM.

**Comment by iizukak** [comment]
@yd8534976 Need your approval.

**Pull request merged**